lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/lib/factory.js

```diff
@@ -23,7 +23,24 @@
 var constantFunction = require( '@stdlib/utils/constant-function' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var exp = require( '@stdlib/math/base/special/exp' );
-var pow = require( '@stdlib/math/base/special/pow' );
+var expm1 = require( '@stdlib/math/base/special/expm1' );
+
+
+// FUNCTIONS //
+
+/**
+* Helper function for repeated computation in the MGF formula.
+*
+* @private
+* @param {number} x - input value
+* @returns {number} evaluated result
+*/
+function phi2( x ) {
+	if ( x === 0 ) {
+		return 1;
+	}
+	return ( 2 * ( expm1( x ) - x ) ) / ( x * x );
```

Comment:
```suggestion
	if ( x === 0.0 ) {
		return 1.0;
	}
	return ( 2.0 * ( expm1( x ) - x ) ) / ( x * x );
```

`phi2` generally expects and returns doubles. As such, similar to C, we add decimals to indicate floating-point values.